backend/src/index.ts

```diff
@@ -87,23 +87,24 @@ async function main() {
       const relayParams = {
         logger,
         target,
-        batchBytesLimit: BATCH_BYTES_LIMIT,
         httpApi,
+        batchBytesLimit: BATCH_BYTES_LIMIT,
       };
 
       let relay;
 
       // Relay accepts different parameters depending on the current mode: process blocks from archive or over the network
       if (chainConfig.downloadedArchivePath) {
         const archive = new ChainArchive({ path: chainConfig.downloadedArchivePath, logger });
-        relay = new Relay({ ...relayParams, archive, batchCountLimit: ARCHIVE_BATCH_COUNT_LIMIT });
+        relay = new Relay({ ...relayParams, batchCountLimit: ARCHIVE_BATCH_COUNT_LIMIT });
```

Comment:
This and `else` branch are identical now, `relay` can be created as `const` above instead.

Reply:
`else` uses `RPC_BATCH_COUNT_LIMIT` instead of `ARCHIVE_BATCH_COUNT_LIMIT`